### PR TITLE
LPS-70677

### PR DESCRIPTION
**Pull request by Ithildir**

### CODE REVIEW

**Comment by brianchandotcom** [comment]
Merged. Thx.

**Comment by brianchandotcom** [comment]
@Ithildir please resend changelog (couldn't CP)

@codyhoag see readme change in just this commit 9e8a9db253281274d32236c22d8bc3bd2e0c2a47

File: modules/sdk/gradle-plugins-defaults/CHANGELOG.markdown [hunk 1]
```diff
@@ -1053,10 +1053,21 @@ files.
...
-- [LPS-70494]: Update the [Liferay Gradle Plugins] dependency to version 3.2.4.
+
+## 2.4.34 - 2017-02-15
+
+### Changed
+- [LPS-70677]: Update the [Liferay Gradle Plugins] dependency to version 3.2.5.
+
+### Removed
+- [LPS-70677]: Remove exclusion of `com.liferay.portal` transitive dependencies
+from the `com.liferay.jasper.jspc` default dependency.
```
Comment (codyhoag) [inline modules/sdk/gradle-plugins-defaults/CHANGELOG.markdown]: @Ithildir Is the `com.liferay.jasper.jspc` dependency here in the `jspC` configuration? We specify it being in the `jspC` config for `gradle-plugins` and `gradle-plugins-jasper-jspc`, but not here. Just wanted to double check. Thanks!

Comment (Ithildir) [inline modules/sdk/gradle-plugins-defaults/CHANGELOG.markdown]: Yeah, you're right! It's the same 😊

**Pull request closed without merging**